interface_frontend.py

```diff
@@ -6,9 +6,12 @@
 MAX_LIVES_COUNT = 3
 MAX_FLAGS_COUNT = 8
 MAX_DAMAGES_COUNT = 9
+TILE_WIDTH = 64
```

Comment:
Vsimla jsem si, ze stejne konstanty mame v modulu `frontend`. A ze modul `frontend` importuje z `interface_frontendu` funkci `get_label`. Mam pocit, ze by ty moduly nemely na sobe zaviset, ale zaroven nechceme kopirovat kod. Nemely bychom tyto konstanty a get_label presunout do utilu nebo noveho modulu pro pomocne funkce pro frontend?

Reply:
to já vůbec netuším. Jakože nechápu jak se importují konstanty. Get_label klidně přesunout, to je mi jasné.

Reply:
Mohlo by postacit `from frontend import TILE_WIDTH`. Jen w tomto pripade bychom se dostaly do spatne situace, protoze ve frontendu importujeme z interface_frontendu, a v ted bychom pridaly import z frontendu do interface_frontendu - coz by se mohlo zacyklit a explodovat ;). Proto me napadlo odklonit spolecne prvky do dalsiho pomocneho modulu, z nehoz budeme importovat do obou frontendu.

Reply:
@befeleme  tak jsem to konečně dodělala. Jen jsem nevěděla, jak ten pomocný modul pojmenovat :-D

Reply:
Kdyz pro backend mame `util`, co treba ho prejmenovat na `util_backend` a stejne tak vytvorit `util_frontend`, abychom se drzely stejne konvence? @ivet1987

Reply:
jo super, hned udělám :-)